audits/rkm0959.md

```diff
@@ -18,7 +18,7 @@ fn duplexing(&mut self, builder: &mut Builder<C>) {
     }
 ```
 
-We are using the full sponge state for the output buffer. This is not an usual behavior for sponge constructions - only the rate part of the sponge state is used for the output usually. One possibility this opens up is that for valid hash result after a single permutation, it is easy to get the hash preimage. Note that this doesn't mean that it's easy to get the hash preimage for any desired hash output result, leading to the low severity.
+We are using the full sponge state for the output buffer. This is not a usual behavior for sponge constructions - only the rate part of the sponge state is used for the output usually. One possibility this opens up is that for valid hash result after a single permutation, it is easy to get the hash preimage. Note that this doesn't mean that it's easy to get the hash preimage for any desired hash output result, leading to the low severity.
```

Comment:
I would even go with 'the' instead of 'a', or just drop the article completely.

Reply:
@matthiasgoergens  Thank you for pointing this out. Removing the underscore was a mistake. I'll revert it to the original format as it is indeed part of the variable name.